app/components/NoticeTypeCheckboxes.tsx

```diff
@@ -157,29 +159,137 @@ const NoticeTypes = {
   ],
 }
 
-interface NoticeTypeCheckboxeProps {
+type DataRow = {
+  topic: string
+  latestRate: number
+}
+
+type FormattedData = {
+  text: DataRow[]
+  binary: DataRow[]
+  voevent: DataRow[]
+}
+
+interface NoticeTypeCheckboxProps {
   defaultSelected?: string[]
+  selectedFormat?: 'binary' | 'text' | 'voevent'
+  showTotalSelected?: boolean
```

Comment:
I don't think that there is a situation where we wouldn't want to show the total selected.

Reply:
Do we want to show it in the quickstart? I added the option so that it could be off for that section, but I can remove the option and display it there too